pkg/naming/names.go

```diff
@@ -100,6 +100,16 @@ func IndexFromName(n string) (int32, error) {
 	return int32(index), nil
 }
 
+func RackOrdinalFromName(name string, sc *scyllav1.ScyllaCluster) (int, error) {
+	for i := range sc.Spec.Datacenter.Racks {
+		if sc.Spec.Datacenter.Racks[i].Name == name {
+			return i, nil
+		}
+	}
```

Comment:
nit: a little bit shorter
```suggestion
	for i, r := range sc.Spec.Datacenter.Racks {
		if r.Name == name {
			return i, nil
		}
	}
```

Reply:
Shorter, but the copy here is a bit unnecessary?

Reply:
I'd lean to readability first, compiler's job is to optimize the copy if it's not needed (+ the N is totally negligible)

Reply:
fair enough, done